bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -40,23 +41,19 @@ import (
 	type_v1 "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
-	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
```

Comment:
this would be ks8s.io/client-go/v2/tools/clientcmd/api until we can remove the /tmp/v2 code - which would be after ksonnet is totally removed.